jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/jar/JarVerifier.java

```diff
@@ -140,13 +146,22 @@ public void beginEntry(JarEntry je, ManifestEntryVerifier mev)
                     return;
                 }
 
+                if (uname.equals(JarFile.MANIFEST_NAME) ||
+                        uname.equals(INDEX_NAME)) {
```

Comment:
Comment that this is J2ObjC changed (from reading `JarIndex.INDEX_NAME` to J2ObjC-defined `INDEX_NAME` above).